packages/@angular/cli/blueprints/ng/files/tslint.json

```diff
@@ -42,16 +43,21 @@
     ],
     "no-construct": true,
     "no-debugger": true,
+    "no-duplicate-super": true,
     "no-duplicate-variable": true,
     "no-empty": false,
     "no-empty-interface": true,
     "no-eval": true,
+    "no-import-side-effect": [true, { "ignore-pattern": "/rxjs.*/" }],
```

Comment:
I think we shouldn't add `no-import-side-effect` on new projects as well. There was a fair bit of back of forth on that rule and it's debatable + needs to be maintained. This can be frustrating for new users.

If rxjs changes as well, then projects made with this version of the CLI would need to change the rule.